src/python/operations/register_optype.cpp

```diff
@@ -30,6 +30,7 @@ void registerOptype(py::module& m) {
       .value("rz", qc::OpType::RZ)
       .value("swap", qc::OpType::SWAP)
       .value("iswap", qc::OpType::iSWAP)
+      //      .value("iswapdg", qc::OpType::iSWAPdg)
```

Comment:
Should be un-commented. There is no reason to not expose this gate to Python.

Reply:
Done ✅